src/DynamoCore/DynamoCore.csproj

```diff
@@ -380,43 +379,40 @@ limitations under the License.
   <ItemGroup>
     <Resource Include="BuiltInAndOperators\BuiltIn.Migrations.xml" />
   </ItemGroup>
-  <ItemGroup />
   <Import Project="$(MSBuildToolsPath)\Microsoft.CSharp.targets" />
   <Target Name="BeforeBuild">
     <ItemGroup>
-      <ExternProtoGeometry Include="$(SolutionDir)..\extern\ProtoGeometry\*" />
-      <ExternProtoGeometryXml Include="$(SolutionDir)..\extern\ProtoGeometry\ProtoGeometry.XML" />
-      <ExternProtoGeometryUICulture Include="$(SolutionDir)..\extern\ProtoGeometry\$(UICulture)\*" />
-      <ExternProtoGeometryLibGLocale Include="$(SolutionDir)..\extern\ProtoGeometry\libg_locale\**\*.*" />
-      <ExternLibG224 Include="$(SolutionDir)..\extern\LibG_224_0_1\*" />
-      <ExternLibG2244 Include="$(SolutionDir)..\extern\LibG_224_4_0\*" />
-      <ExternLibG225 Include="$(SolutionDir)..\extern\LibG_225_0_0\*" />
-      <ExternLibG226 Include="$(SolutionDir)..\extern\LibG_226_0_0\*" />
-      <ExternLibG Include="$(SolutionDir)..\extern\LibG_*\*" />
+      <LibGInterface Include="$(SolutionDir)\packages\DynamoVisualProgramming.LibG_226_0_0*\tools\net48\LibG.Interface.dll" />
```

Comment:
What does the `*` wildcard do here?

Reply:
It takes all the files under that subfolder

On the line 395 when the CI pipeline creates the folder on runtime, on this case "extern/LibG_225_0_0" it will copy all the contents of that folder to the bin/Release/LibG_225_0_0. On the future, on the switch to 226, we won't need to change that line